data/actionsmap/yunohost.yml

```diff
@@ -925,6 +925,54 @@ monitor:
 
 
 #############################
+#         Settings          #
+#############################
+settings:
+    category_help: Manage YunoHost global settings
+    actions:
+
+        ### settings_list()
+        list:
+            action_help: list all entries of the settings
+            api: GET /settings
+
+        ### settings_get()
+        get:
+            action_help: get an entry value in the settings
+            api: GET /settings/<key>
+            arguments:
+                key:
+                    help: Settings key
+
+        ### settings_set()
+        set:
+            action_help: set an entry value in the settings
+            api: POST /settings/<key>
+            arguments:
+                key:
+                    help: Settings key
+                -v:
+                    full: --value
+                    help: new value
+                    extra:
+                        required: True
+                -n:
+                    full: --namespace
+                    help: 'The namespace in which the settings is stored (eg: "ssl"))'
+                    extra:
+                        required: True
+
+        ### settings_reset()
+        reset:
+            action_help: reset all settings to their default value
+            api: POST /settings/reset
+            arguments:
+                -y:
+                    full: --yes
+                    help: Yes I'm sure I want to do that
+                    action: store_true
+
```

Comment:
I think we miss the ability to reset a specific setting ( `DELETE /settings/<key>` )